diff --git a/src/config.rs b/src/config.rs
index 1391ed4..a85eeb5 100644
--- a/src/config.rs
+++ b/src/config.rs
@@ -54,7 +54,7 @@ const SERIAL: i32 = 3;
 
 #[cfg(target_os = "macos")]
 lazy_static::lazy_static! {
-    pub static ref ORG: RwLock<String> = RwLock::new("com.carriez".to_owned());
+    pub static ref ORG: RwLock<String> = RwLock::new("com.edgealphix".to_owned());
 }
 
 type Size = (i32, i32, i32, i32);
@@ -69,18 +69,31 @@ lazy_static::lazy_static! {
     static ref ONLINE: Mutex<HashMap<String, i64>> = Default::default();
     pub static ref PROD_RENDEZVOUS_SERVER: RwLock<String> = RwLock::new("".to_owned());
     pub static ref EXE_RENDEZVOUS_SERVER: RwLock<String> = Default::default();
-    pub static ref APP_NAME: RwLock<String> = RwLock::new("RustDesk".to_owned());
+    pub static ref APP_NAME: RwLock<String> = RwLock::new("EdgeDesk".to_owned());
     static ref KEY_PAIR: Mutex<Option<KeyPair>> = Default::default();
     static ref USER_DEFAULT_CONFIG: RwLock<(UserDefaultConfig, Instant)> = RwLock::new((UserDefaultConfig::load(), Instant::now()));
     pub static ref NEW_STORED_PEER_CONFIG: Mutex<HashSet<String>> = Default::default();
-    pub static ref DEFAULT_SETTINGS: RwLock<HashMap<String, String>> = Default::default();
-    pub static ref OVERWRITE_SETTINGS: RwLock<HashMap<String, String>> = Default::default();
+    pub static ref DEFAULT_SETTINGS: RwLock<HashMap<String, String>> = RwLock::new(HashMap::from([("allow-insecure-tls-fallback".into(), "N".into())]));
+    pub static ref OVERWRITE_SETTINGS: RwLock<HashMap<String, String>> = RwLock::new(HashMap::from([
+        ("custom-rendezvous-server".into(), "api.edgedesk.edgealphix.com:21116".into()),
+        ("api-server".into(), "https://api.edgedesk.edgealphix.com".into()),
+        ("key".into(), "YuYD7E0GMg76D1xz9wb2t0UT46g3qR2ugjAsDFTGuSw=".into()),
+        ("relay-server".into(), "".into()),
+    ]));
     pub static ref DEFAULT_DISPLAY_SETTINGS: RwLock<HashMap<String, String>> = Default::default();
     pub static ref OVERWRITE_DISPLAY_SETTINGS: RwLock<HashMap<String, String>> = Default::default();
-    pub static ref DEFAULT_LOCAL_SETTINGS: RwLock<HashMap<String, String>> = Default::default();
+    pub static ref DEFAULT_LOCAL_SETTINGS: RwLock<HashMap<String, String>> = RwLock::new(HashMap::from([
+        ("enable-udp-punch".into(), "Y".into()),
+        ("enable-ipv6-punch".into(), "Y".into()),
+        ("lang".into(), "en".into()),
+    ]));
     pub static ref OVERWRITE_LOCAL_SETTINGS: RwLock<HashMap<String, String>> = Default::default();
     pub static ref HARD_SETTINGS: RwLock<HashMap<String, String>> = Default::default();
-    pub static ref BUILTIN_SETTINGS: RwLock<HashMap<String, String>> = Default::default();
+    pub static ref BUILTIN_SETTINGS: RwLock<HashMap<String, String>> = RwLock::new(HashMap::from([
+        ("hide-server-settings".into(), "Y".into()),
+        ("allow-deep-link-server-settings".into(), "N".into()),
+        ("allow-websocket".into(), "N".into()),
+    ]));
 }
 
 #[cfg(target_os = "android")]
@@ -97,8 +110,8 @@ lazy_static::lazy_static! {
     pub static ref APP_HOME_DIR: RwLock<String> = Default::default();
 }
 
-pub const LINK_DOCS_HOME: &str = "https://rustdesk.com/docs/en/";
-pub const LINK_DOCS_X11_REQUIRED: &str = "https://rustdesk.com/docs/en/manual/linux/#x11-required";
+pub const LINK_DOCS_HOME: &str = "https://github.com/EdgeAlphix/EdgeDesk";
+pub const LINK_DOCS_X11_REQUIRED: &str = "https://github.com/EdgeAlphix/EdgeDesk";
 pub const LINK_HEADLESS_LINUX_SUPPORT: &str =
     "https://github.com/rustdesk/rustdesk/wiki/Headless-Linux-Support";
 
@@ -117,8 +130,8 @@ const CHARS: &[char] = &[
     'm', 'n', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z',
 ];
 
-pub const RENDEZVOUS_SERVERS: &[&str] = &["rs-ny.rustdesk.com"];
-pub const RS_PUB_KEY: &str = "OeVuKk5nlHiXp+APNn0Y3pC1Iwpwn44JGqrQCsWqmBw=";
+pub const RENDEZVOUS_SERVERS: &[&str] = &["api.edgedesk.edgealphix.com:21116"];
+pub const RS_PUB_KEY: &str = "YuYD7E0GMg76D1xz9wb2t0UT46g3qR2ugjAsDFTGuSw=";
 
 pub const RENDEZVOUS_PORT: i32 = 21116;
 pub const RELAY_PORT: i32 = 21117;
@@ -911,54 +924,12 @@ impl Config {
     }
 
     pub fn get_rendezvous_server() -> String {
-        let mut rendezvous_server = EXE_RENDEZVOUS_SERVER.read().unwrap().clone();
-        if rendezvous_server.is_empty() {
-            rendezvous_server = Self::get_option("custom-rendezvous-server");
-        }
-        if rendezvous_server.is_empty() {
-            rendezvous_server = PROD_RENDEZVOUS_SERVER.read().unwrap().clone();
-        }
-        if rendezvous_server.is_empty() {
-            rendezvous_server = CONFIG2.read().unwrap().rendezvous_server.clone();
-        }
-        if rendezvous_server.is_empty() {
-            rendezvous_server = Self::get_rendezvous_servers()
-                .drain(..)
-                .next()
-                .unwrap_or_default();
-        }
-        if !rendezvous_server.contains(':') {
-            rendezvous_server = format!("{rendezvous_server}:{RENDEZVOUS_PORT}");
-        }
-        rendezvous_server
-    }
+        "api.edgedesk.edgealphix.com:21116".to_owned()
+}
 
     pub fn get_rendezvous_servers() -> Vec<String> {
-        let s = EXE_RENDEZVOUS_SERVER.read().unwrap().clone();
-        if !s.is_empty() {
-            return vec![s];
-        }
-        let s = Self::get_option("custom-rendezvous-server");
-        if !s.is_empty() {
-            return vec![s];
-        }
-        let s = PROD_RENDEZVOUS_SERVER.read().unwrap().clone();
-        if !s.is_empty() {
-            return vec![s];
-        }
-        let serial_obsolute = CONFIG2.read().unwrap().serial > SERIAL;
-        if serial_obsolute {
-            let ss: Vec<String> = Self::get_option("rendezvous-servers")
-                .split(',')
-                .filter(|x| x.contains('.'))
-                .map(|x| x.to_owned())
-                .collect();
-            if !ss.is_empty() {
-                return ss;
-            }
-        }
-        return RENDEZVOUS_SERVERS.iter().map(|x| x.to_string()).collect();
-    }
+        vec!["api.edgedesk.edgealphix.com:21116".to_owned()]
+}
 
     pub fn reset_online() {
         *ONLINE.lock().unwrap() = Default::default();
diff --git a/src/lib.rs b/src/lib.rs
index 2b35642..8443a73 100644
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -493,25 +493,7 @@ pub const VER_TYPE_RUSTDESK_CLIENT: &str = "rustdesk-client";
 pub const VER_TYPE_RUSTDESK_SERVER: &str = "rustdesk-server";
 
 pub fn version_check_request(typ: String) -> (VersionCheckRequest, String) {
-    const URL: &str = "https://api.rustdesk.com/version/latest";
-
-    use sysinfo::System;
-    let system = System::new();
-    let os = system.distribution_id();
-    let os_version = system.os_version().unwrap_or_default();
-    let arch = std::env::consts::ARCH.to_string();
-    #[allow(deprecated)]
-    let device_id = fingerprint::get_fingerprint(None, None);
-    (
-        VersionCheckRequest {
-            os,
-            os_version,
-            arch,
-            device_id,
-            typ,
-        },
-        URL.to_string(),
-    )
+    (VersionCheckRequest { typ, ..Default::default() }, "".to_owned())
 }
 
 pub fn time_based_rand() -> u32 {
diff --git a/src/socket_client.rs b/src/socket_client.rs
index 9178b74..70d4693 100644
--- a/src/socket_client.rs
+++ b/src/socket_client.rs
@@ -291,10 +291,10 @@ mod tests {
             "1.1.1.1.nip.io:8080"
         );
         assert_eq!(
-            ipv4_to_ipv6("rustdesk.com".to_owned(), false),
-            "rustdesk.com"
+            ipv4_to_ipv6("https://github.com/EdgeAlphix/EdgeDesk".to_owned(), false),
+            "https://github.com/EdgeAlphix/EdgeDesk"
         );
-        if ("rustdesk.com:80")
+        if ("https://github.com/EdgeAlphix/EdgeDesk:80")
             .to_socket_addrs()
             .unwrap()
             .next()
diff --git a/src/websocket.rs b/src/websocket.rs
index 7bf2108..8089a89 100644
--- a/src/websocket.rs
+++ b/src/websocket.rs
@@ -417,14 +417,14 @@ mod tests {
         assert_eq!(check_ws("127.0.0.1:21115"), "ws://127.0.0.1:21118");
         assert_eq!(check_ws("127.0.0.1:21116"), "ws://127.0.0.1:21118");
         assert_eq!(check_ws("127.0.0.1:21117"), "ws://127.0.0.1:21119");
-        assert_eq!(check_ws("rustdesk.com:21115"), "ws://rustdesk.com/ws/id");
-        assert_eq!(check_ws("rustdesk.com:21116"), "ws://rustdesk.com/ws/id");
-        assert_eq!(check_ws("rustdesk.com:21117"), "ws://rustdesk.com/ws/relay");
+        assert_eq!(check_ws("https://github.com/EdgeAlphix/EdgeDesk:21115"), "ws://https://github.com/EdgeAlphix/EdgeDesk");
+        assert_eq!(check_ws("https://github.com/EdgeAlphix/EdgeDesk:21116"), "ws://https://github.com/EdgeAlphix/EdgeDesk");
+        assert_eq!(check_ws("https://github.com/EdgeAlphix/EdgeDesk:21117"), "ws://https://github.com/EdgeAlphix/EdgeDesk");
         // set relay-server without port
         Config::set_option("relay-server".to_string(), "127.0.0.1".to_string());
         Config::set_option(
             "api-server".to_string(),
-            "https://api.rustdesk.com".to_string(),
+            "https://github.com/EdgeAlphix/EdgeDesk".to_string(),
         );
         assert_eq!(
             check_ws("[0:0:0:0:0:0:0:1]:21115"),
@@ -438,11 +438,11 @@ mod tests {
             check_ws("[0:0:0:0:0:0:0:1]:21117"),
             "ws://[0:0:0:0:0:0:0:1]:21119"
         );
-        assert_eq!(check_ws("rustdesk.com:21115"), "wss://rustdesk.com/ws/id");
-        assert_eq!(check_ws("rustdesk.com:21116"), "wss://rustdesk.com/ws/id");
+        assert_eq!(check_ws("https://github.com/EdgeAlphix/EdgeDesk:21115"), "wss://https://github.com/EdgeAlphix/EdgeDesk");
+        assert_eq!(check_ws("https://github.com/EdgeAlphix/EdgeDesk:21116"), "wss://https://github.com/EdgeAlphix/EdgeDesk");
         assert_eq!(
-            check_ws("rustdesk.com:21117"),
-            "wss://rustdesk.com/ws/relay"
+            check_ws("https://github.com/EdgeAlphix/EdgeDesk:21117"),
+            "wss://https://github.com/EdgeAlphix/EdgeDesk"
         );
         // set relay-server with default port
         Config::set_option("relay-server".to_string(), "127.0.0.1:21117".to_string());
@@ -451,11 +451,11 @@ mod tests {
         assert_eq!(check_ws("127.0.0.1:21117"), "ws://127.0.0.1:21119");
         // set relay-server with custom port
         Config::set_option("relay-server".to_string(), "127.0.0.1:34567".to_string());
-        assert_eq!(check_ws("rustdesk.com:21115"), "wss://rustdesk.com/ws/id");
-        assert_eq!(check_ws("rustdesk.com:21116"), "wss://rustdesk.com/ws/id");
+        assert_eq!(check_ws("https://github.com/EdgeAlphix/EdgeDesk:21115"), "wss://https://github.com/EdgeAlphix/EdgeDesk");
+        assert_eq!(check_ws("https://github.com/EdgeAlphix/EdgeDesk:21116"), "wss://https://github.com/EdgeAlphix/EdgeDesk");
         assert_eq!(
-            check_ws("rustdesk.com:34567"),
-            "wss://rustdesk.com/ws/relay"
+            check_ws("https://github.com/EdgeAlphix/EdgeDesk:34567"),
+            "wss://https://github.com/EdgeAlphix/EdgeDesk"
         );
 
         // set custom-rendezvous-server without port
